README.md

```diff
@@ -1,21 +1,21 @@
 # isaac-api
 
-![Java CI with Maven](https://github.com/isaacphysics/isaac-api/workflows/Java%20CI%20with%20Maven/badge.svg?branch=master)
+![Java CI with Maven](https://github.com/isaaccomputerscience/isaac-api/workflows/Java%20CI%20with%20Maven/badge.svg?branch=master)
```

Comment:
I believe the CS repo is using the newer convention of `main` as the primary branch rather than `master`. You may want to point at that one for this?